### PR TITLE
Misc libvirt fixes

### PR DESCRIPTION
**Pull request by javacruft**
Using the libvirt interface to interact with a deb installed
libvirt from within snaps using the python-libvirt library
required a few misc updates to ensure that the library can
access /etc/libvirt/*, and can use the listen and accept
syscalls when connecting to the libvirt socket.

Without these updates, python processes using python-libvirt
switch to defunct state on accessing the libvirt socket.

### CODE REVIEW

**Comment by javacruft** [comment]
This commit is stacked ontop of my openvswitch-support interface changes.

File: interfaces/builtin/libvirt.go [hunk 1]
```diff
@@ -35,6 +36,8 @@ sendto
 sendmsg
 socket
 socketpair
+listen
+accept
```
Comment (jdstrand) [inline interfaces/builtin/libvirt.go]: This is curious. What in the client (ie, the plugging side) is setting up a listener?

Comment (javacruft) [inline interfaces/builtin/libvirt.go]: I need to dig into that more - without these stanza's the python process goes defunct on opening the connection to libvirt.

**Comment by javacruft** [comment]
Raised combined pull request for all openstack related changes - closing this one

**Pull request closed without merging**